frontend/src/app/modules/home/home-page/home-page.component.html

```diff
@@ -5,83 +5,100 @@
             <img alt="logo" src="assets/logo-white.svg">
         </div>
         <div *ngIf="!user" class="user-info">
-            <button pButton pRipple type="button" label="Sign in" class="p-button-outlined sign-in-button"></button>
+            <button class="p-button-outlined sign-in-button" label="Sign in" pButton pRipple type="button"></button>
         </div>
         <div *ngIf="user" class="user-info">
-            <img class="user-avatar" [src]="user.avatarUrl ?? 'assets\\avatar.png'" alt="Avatar">
+            <img [src]="user.avatarUrl ?? 'assets\\avatar.png'" alt="Avatar" class="user-avatar">
             <div (click)="menu.toggle($event)" class="user-name">{{user.firstName}} {{user.lastName}}</div>
-            <p-menu #menu [popup]="true" [model]="userItems"></p-menu>
+            <p-menu #menu [model]="userItems" [popup]="true"></p-menu>
         </div>
     </div>
 
-    <div class="sidebar">
-        <app-organization-menu class="sidebar-avatar"></app-organization-menu>
+    <div [ngStyle]="{'width': !collapsed ? '230px' : '68px'}" class="sidebar">
```

Comment:
Please don't use inline styles. You can use 2 different classes with different width and will swap them if you need.

Reply:
done